### PR TITLE
Norton account supports U2F

### PR DESCRIPTION
**Pull request by tprasadtp**
Norton account now supports U2F keys. This is different than [Yubikey VIP](https://www.yubico.com/why-yubico/for-business/authentication-solutions/symantec/), normal u2f only Yubikey works too.

### CODE REVIEW

**Comment by techge** [comment]
Do you have any resource that supports that it is really hardware 2fa, I couldn't find it without being forced to log in. Do you know if they support OTP, U2F or both?

Could you please update the _data/devices/security.yml accordingly? This is the file which is actually shown on website...

Thx!

**Comment by tprasadtp** [comment]
Hi,
I have updated with the link which should give more information on that. I have also updated the file in devices/security.yml 
and also, here are few screenshots.
image

image

**Comment by techge** [comment]
Thanks a lot for your RP! Looks good.
Last problem I see... is there really general OTP support? In the docs you attached there is shown FIDO for sure, and the VIP Access App which seems to do the same as Google authenticator and other OTP Apps. 
screenshot_20180129_103718

Just because I do not know the process for sure, will one get the secret code when registering for VIP Access App? If not there is no use to mark it as compatible for OTP from a hardware perspective as you will only be able to use it with the proprietary App.

You see what I mean? I just don't know this VIP Access App and probably can't test without an account, can I?
Sorry, that this is bit trickier in this case!

**Comment by tprasadtp** [comment]
The VIP access app uses same algorithm as Google Authenticator, but the provisioning is different. VIP Server keeps record of a serial number associated with every seed. So, provisioning of token has to happen with a HTTP request to VIP servers and same VIP Credential ID can be used on multiple  services. Though technically it is possible to use Google Authentiacator/ Authy or any other TOTP APP (Even HOTP if you send right parameters with the POST request) the official way to use TOTP is to use VIP App on your device. (Windows, MacOS, Android and iOS are supported)
* To test the VIP Access app you do not need to login or have to Norton account or any account you just need to "Provision the token" from the app. There is link on their [homepage](https://idprotect.vip.symantec.com/) to test the token.
* One cannot get the seed while registering the app. It stays hidden unless you use unofficial python client which can get the seed for you.
* I have verified that the seed if obtained, works with Google Authenticator.
* FIDO Secure key is separate. It uses U2F protocol.
TLDR, 
- Yes, one cannot get the Secret/Seed while registering a  token unless special steps are taken.
- One HAS to use VIP Access App to generate the TOTP codes if seed is not obtained.
- There is special Yubikey VIP which comes pre-configured with VIP tokens and can be used as well. However its uses HOTP token and it predates the U2F protocol.

Perhaps I can add an exception saying it requires additional app to use TOTP? Because Normal end users are unlikely to use the unofficial VIP client to get the seeds.

**Comment by techge** [comment]
Thank you very much for this very detailed explanation! 
Imho it is to complicated for general users to use OTP functions with a dongle. And this is what dongleauth.info is about, isn't it? Therefore I would prefer to mark it as not supporting OTP, but using the exception field to make clear, that one *could* use OTP if taking specials steps. Are you okay with this solution?

**Comment by tprasadtp** [comment]
My apologies for the confusion, But Norton accounts do support TOTP. Its just that if you need to get the seeds aka secret you need to use special methods. Otherwise Just installing VIP Access app is enough. 

From what I can see from other examples in [here](http://www.dongleauth.info/#domains) some cases of TOTP require special app. As in the page I linked previously, Cloudflare doesn't provide the seeds either. We have to use Authy app to get TOTP tokens( [more info here](https://support.cloudflare.com/hc/en-us/articles/200167866-What-is-Authy-2-Factor-Authentication)).  And so is the case for Microsoft Account if we use their default choice of app while setting up 2FA. May be I am interpreting it wrong, but doesn't
```
otp: Yes
``` 
mean that OTP is supported regardless if its app based or the secret lives in a hardware key? I am basing view on other examples, please let me know if its not the case. 

The contributing page however does mention 

>There are many forms of Two Factor Auth, but DongleAuth.info is only interested in listing sites that support Two Factor Authentication using USB dongles. Currently that means the site must support either One Time Passwords (HOTP / RFC 4226 or TOTP / RFC 6238) or FIDO Universal 2nd Factor (U2F).

But there are many entries like Microsoft, Cloudflare shouldn't be there as they cant be used with hardware. 

So my question is, If I cannot get the seed/secret by official methods,  should I put otp as no ? If So, then Yes, I agree with your point completely.  I have already made the changes you requested and will push it after getting this clarification.

**Comment by techge** [comment]
But the seed is seen on the screenshots of the cloudflare site you have linked. As a rule of thumb, if someone can use Google Authenticator, one can easily get the seed for different apps as well. In best case the seed is shown directly (as in the screenshots, "backup code") or at least as a qr-code which represents the seed.

Does the Norton Account give us a seed this way? If not, yes I would be happy for the PR. dongleauth.info should really just include otp which can actually be used with hardware dongles without to much hacking. If you found examples which don't, please open a issue or PR or just tell us.

img_3701

**Comment by tprasadtp** [comment]
Hi, 
I have  removed otp as VIP access doesnot give seeds officially anyway. Thanks for clarifying.
